Orange/widgets/unsupervised/owsom.py

```diff
@@ -824,17 +825,20 @@ def update_output(self):
                     rows = self.get_member_indices(x, y)
                     indices[rows] = self.selection[x, y]
 
+        new_data = self.data.add_column(ContinuousVariable(name='som-x'), self.coordinates[:, 0], True)
```

Comment:
A variable with this name may already exist in the data. You should use

```
name_x, name_y = get_unique_names(self.data.domain, ("som-x", "som-y"))
```

which will append numbers (e.g. som-x (3), som-y (3)) if needed.